### PR TITLE
local-tester: procfile, faults, and network bridge

### PR DESCRIPTION
**Pull request by heyitsanthony**
Creates a local fault injected cluster and stresser for etcd.

Usage: goreman -f tools/local-tester/Procfile start

### CODE REVIEW

**Comment by heyitsanthony** [comment]
Run it for long enough and there's some wal corruption

File: tools/local-tester/bridge/bridge.go [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+	outconn, err := net.Dial("tcp", os.Args[2])
+	if err != nil {
+		log.Println("oops:", err)
+		return
+	}
+	log.Printf("one way %v -> %v\n", outconn.LocalAddr(), outconn.RemoteAddr())
+	io.Copy(outconn, conn)
+}
+
+func randCopy(conn net.Conn, outconn net.Conn) {
```
Comment (xiang90) [inline tools/local-tester/bridge/bridge.go]: this is violent...

**Comment by xiang90** [comment]
lgtm. probably add a little readme file for how to build and run this?

**Comment by gyuho** [comment]
Yeah README would be helpful. LGTM. Thanks.

**Comment by xiang90** [comment]
LGTM

**Pull request merged**